modules/integration/tests-integration/tests-resources/tests-resource-registry/src/test/java/org/wso2/greg/integration/resources/server/mgt/test/RegistryConfiguratorTestCase.java

```diff
@@ -194,6 +196,19 @@ private void increaseSearchIndexStartTimeDelay () throws Exception {
             AXIOMXPath xpathExpression3 = new AXIOMXPath("/wso2registry/indexingConfiguration/indexerPoolSize");
```

Comment:
Hi Imesh,
Thanks for pointing this out. But this variable is initialized inside try block, since there can be exceptions when getting values from the registry.xml.
Thanks!